### DISCUSSION TITLE
How do I link images?

### DISCUSSION THREAD

**Discussion by artisanphil** [goravel/goravel#458] [Q&A] [answered]
I assume the place to put images is inside the public folder. So I created a folder `images` inside `public` and added a file test.png there. Now in Laravel I would be able to link it in my blade file like this: `<img src="/images/test.png" />`, but I'm not able to link to the file like that in the tmpl file or access it directly through the url in the browser. Where should I place images and how can they be made accessible?

**Comment by hwbrzzl** [comment, marked as answer]
You need to create a file routing: https://www.goravel.dev/the-basics/routing.html#file-routing

**Reply by artisanphil** [reply]
Thanks @hwbrzzl That works. For others, I added this to web.go:
```
facades.Route().StaticFile("persons/1.png", "./public/persons/1.png")
```

Is there a way (maybe with regex) to map all files in a folder?

**Reply by hwbrzzl** [reply]
You can use `Static` to route a folder.
